hrjobcontract/js/test/services/contract-pension_test.js

```diff
@@ -0,0 +1,60 @@
+define([
+  'common/angular',
+  'common/angularMocks',
+  'job-contract/app'
+], function(angular, Mock) {
+  'use strict';
+
+  describe('ContractPensionService', function () {
+    var $httpBackend, responseData, $scope, ContractPensionService;
+
+    beforeEach(module('hrjc'));
+    beforeEach(inject(function (_ContractPensionService_, _$httpBackend_, _$rootScope_) {
+      ContractPensionService = _ContractPensionService_;
+      $httpBackend = _$httpBackend_;
+      $scope = _$rootScope_;
+
+      responseData = {
+        "is_error": 0,
+        "undefined_fields": [
+          "jobcontract_revision_id"
+        ],
+        "version": 3,
+        "count": 1,
+        "id": 46,
+        "values": [
+          {
+            "id": "46",
+            "jobcontract_revision_id": "68"
+          }
+        ],
+        "xdebug": {
+          "peakMemory": 57660648,
+          "memory": 57479304,
+          "timeIndex": 1.44244098663
+        }
+      };
+
+      $httpBackend.when('GET','/index.php?q=civicrm/ajax/rest&action=get&entity=HRJobContract&json=%7B%22sequential%22:1,%22deleted%22:0%7D').respond(responseData);
+      $httpBackend.when('GET','/index.php?q=civicrm/ajax/rest&action=get&entity=HRJobPension&json=%7B%22jobcontract_revision_id%22:68,%22sequential%22:1,%22debug%22:null%7D').respond(responseData);
+      $httpBackend.whenGET(/views.*/).respond(responseData);
+    }));
+
+    describe('when calling getOne()', function () {
+      it("defines getOne() function", function () {
+        expect(ContractPensionService.getOne).toBeDefined();
+      });
+
+      it('calls getOne fuction and return expected values', function (done) {
+        $httpBackend.expectGET('/index.php?q=civicrm/ajax/rest&action=get&entity=HRJobContract&json=%7B%22sequential%22:1,%22deleted%22:0%7D');
```

Comment:
Can't move flush to beforeEach due to following error:
<img width="986" alt="screen shot 2017-01-31 at 7 20 18 pm" src="https://cloud.githubusercontent.com/assets/6307362/22466897/79f53140-e7ea-11e6-8c74-00deaa34d335.png">